### DISCUSSION TITLE
Can generate command overwrite existing file ?

### DISCUSSION THREAD

**Discussion by MasterMong** [loco-rs/loco#645] [Q&A]
I need help to modify model, CRUD controller by generate command then overwrite existing file.

**Comment by kaplanelad** [comment]
Hey @MasterMong 
If you haven't deployed the migration to production yet, you have two options:

1. Modify the Migration Manually: Adjust the existing migration file and rerun the generated entity.
2. Delete and Regenerate: Delete the current migration file and run the loco generate command with the updated fields.

If the migration is already in production, it's recommended that a new migration script be created to modify the database accordingly.

maybe this seaorm docs will help you
https://www.sea-ql.org/SeaORM/docs/next/migration/writing-migration/

**Reply by kaplanelad** [reply]
See also this related discussion:
https://github.com/loco-rs/loco/discussions/635
